portfolio/src/main/java/com/google/sps/models/Comment.java

```diff
@@ -0,0 +1,67 @@
+package com.google.sps.models;
+
+import java.util.Date;
+
+public class Comment {
+    private String commentText;
+    private long createdAt;
+    private long id;
+
+    public Comment(String commentText, long createdAt, long id) {
+        this.commentText = commentText;
+        this.createdAt = createdAt;
+        this.id = id;
+    }
+
+    public Comment(String commentText) {
+        this.commentText = commentText;
+        this.createdAt = System.currentTimeMillis();
+    }
+
+    public long getId() {
+        return id;
+    }
+
+    public void setId(long id) {
+        this.id = id;
+    }
+
+    public String getCommentText() {
+        return commentText;
+    }
+
+    public void setCommentText(String commentText) {
+        this.commentText = commentText;
+    }
+
+    public long getCreatedAt() {
+        return createdAt;
+    }
+
+    public void setCreatedAt(long createdAt) {
+        this.createdAt = createdAt;
+    }
+
+    public static final class Validator {
```

Comment:
Good use of validator!

Reply:
Yeah. I am planning to return error messages as well through ValidationResponse which can be used on frontend to display errors.